server/src/main/java/io/crate/execution/ddl/tables/AlterTableClient.java

```diff
@@ -139,14 +114,10 @@ public CompletableFuture<Long> addColumn(AddColumnRequest addColumnRequest) {
                     }
                     LOGGER.warn(finalWarning);
                 }
-                return transportAddColumnAction.execute(addColumnRequest).thenApply(_ -> -1L);
+                return client.execute(TransportAddColumn.ACTION, addColumnRequest).thenApply(ignored -> -1L);
```

Comment:
Any reason for the change from `_` to `ignored` ?

jdtls warns about unused lambda parameters:

![image](https://github.com/user-attachments/assets/c4efc994-9849-409f-80a2-e7545a4a5614)

Reply:
intellij warns about `_`, I can revert, and disable this in Intellij.